boards/b-l475e-iot01a/include/periph_conf.h

```diff
@@ -203,6 +203,7 @@ static const i2c_conf_t i2c_config[] = {
         .sda_af         = GPIO_AF4,
         .bus            = APB1,
         .rcc_mask       = RCC_APB1ENR1_I2C1EN,
+        .rcc_sw_mask    = RCC_CCIPR_I2C1SEL_1, /* HSI (16 MHz) */
```

Comment:
With #9728 being merged, this change also needs to be applied to `boards/common/stm32/cfg_i2c1_pb8_pb9.h`